client/cmd/apps.go

```diff
@@ -175,8 +175,8 @@ func printLogs(logs string) error {
 	for _, log := range strings.Split(logs, `\\n\\n`) {
 		category := "unknown"
 		parts := strings.Split(strings.Split(log, " -- ")[0], " ")
-		if len(parts) >= 2 {
-			category = parts[1]
+		if len(parts) >= 1 {
```

Comment:
I don't think this check is needed any more. `strings.Split()` will always return a []string with a length >= 1. This was just to make sure that the string itself had been split at least once.